ui/components/Dataset/index.js

```diff
@@ -9,6 +9,13 @@ import { useQueryContext } from '../../context/query';
 
 const DATE_FORMAT = 'do MMM yyyy';
 
+function Embolden({ children }) {
+  const { getSelections } = useQueryContext();
+  const { q } = getSelections();
+  const re = new RegExp(`(${q})`, 'ig')
+  return <span dangerouslySetInnerHTML={{ __html: children.replace(re, '<b>$1</b>') }} />
```

Comment:
Given that this is being set with user input from the query string, I think this is _actually_ dangerous.

Reply:
`?q=<script>alert('All your base are belong to me');</script>`

Reply:
Just to have a record of this on here: we've concluded this is _probably okay_ given we're only replacing content that exists in data if it matches, so malicious injections won't be matched. We can investigate more at pen test